README.md

```diff
@@ -256,9 +256,50 @@ To configuring a pipeline:
 | [stage](https://docs.gitlab.com/ee/ci/yaml/#stage) | Defines a job stage (E.g., build) |
 | [variables](https://docs.gitlab.com/ee/ci/yaml/#variables) | Define job variables on a job level |
 
-### GitHub Pipelines
+## GitHub 
+
+### Set-up remote 
+
+* Rename current Gitlab remote
+  ```bash
+  $ git remote rename origin gitlab
+  ```
+
+* Update Git to add new remote to GitHub: 
+  ```bash
+  $ git remote add github git@github.com:arzukhanna/sudoku-python.git
+  ```
+
+* Force push main branch from GitLab to GitHub
+  ```bash
+  $ git push -fu github main
+  ```
+  
+* Push develop and feature branch from GitLab to GitHub
+  ```bash
+  $ git push -u github develop
+  $ git push -u github feature
```

Comment:
Not quite: you forgot to git checkout branch first.
The above code could clobber develop with feature or worse clobber main with a feature ....